frontend-react/src/resources/AuthResource.ts

```diff
@@ -20,6 +20,7 @@ export default class AuthResource extends Resource {
                 ...init.headers,
                 Authorization: `Bearer ${accessToken}`,
                 Organization: organization || "",
+                "authentication-type": "okta",
```

Comment:
UI now specifies OktaAuth for submissions queries.  (Previously submissions queries only supported okta, so the additional header flag was not needed.  Now submissions supports two different auth techniques)